logstash-core/lib/logstash/environment.rb

```diff
@@ -1,7 +1,40 @@
 # encoding: utf-8
 require "logstash/errors"
+require "logstash/config/cpu_core_strategy"
+require "logstash/settings"
 
 module LogStash
+
+  [
+            Setting::String.new("node.name", Socket.gethostname),
```

Comment:
the indentation was on purpose, to align the settings names, to make it easier to read.